src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_dropdown.scss

```diff
@@ -73,21 +73,21 @@ A basic dropdown that is currently used for resource actions.
 
 // The dropdown menu (ul)
 .dropdown-menu {
-    border-radius: 3px;
+    border-radius: 2px;
     float: left;
     min-width: 120px;
     padding: 0;
     margin: 5px 0 0;
     background-color: $color-text-inverted;
-    border: 1px solid $color-text-introvert;
+    border: 0.5px solid $color-structure-normal;
```

Comment:
`0.5px` per `li` element inside the menu plus `0.5px` for the `dropdown-menu` itself makes the border exactly `1px` everywhere (I hope).